cmd/influxd/launcher/launcher.go

```diff
@@ -940,11 +940,10 @@ func (m *Launcher) run(ctx context.Context, opts *InfluxdOpts) (err error) {
 		)
 	}
 
-	notebookSvc, err := notebooks.NewService()
```

Comment:
There is no error that can be returned from the `NewService` method, so I deleted the error handling code.